### DISCUSSION TITLE
Integration with WhiteSource vulnerability reporting?

### DISCUSSION THREAD

**Discussion by japj** [renovatebot/renovate#8972] [Request Help] [answered]
Hi, perhaps I missed it in the documentation, but is there a way to integrate renovate with existing WhiteSource vulnerability reporting w.r.t. PR generation?
From the doc it seems vulnerabilityAlerts is only support with GitHub?

**Comment by rarkins** [comment, marked as answer]
This is supported in WhiteSource's commercial product "WhiteSource for Developers". Details are here: https://www.whitesourcesoftware.com/native-integrations-for-developers-environments/
